dashboard/client/src/type/serve.ts

```diff
@@ -92,6 +92,7 @@ export enum ServeSystemActorStatus {
   HEALTHY = "HEALTHY",
   UNHEALTHY = "UNHEALTHY",
   DRAINING = "DRAINING",
+  DRAINED = "DRAINED",
```

Comment:
Do we need to expose `DRAINED`? This will only be a momentary state just before the proxy is shut down.

Reply:
We don't. I added it purely for consistency to keep them in sync. But I guess I can just add a comment and remove it from frontend.